src/Build/Evaluation/Conditionals/Token.cs

```diff
@@ -32,13 +32,26 @@ internal sealed class Token
         /// </summary>
         internal enum TokenType
         {
-            Comma, LeftParenthesis, RightParenthesis,
-            LessThan, GreaterThan, LessThanOrEqualTo, GreaterThanOrEqualTo,
-            And, Or,
-            EqualTo, NotEqualTo, Not,
-            Property, String, Numeric, ItemList, ItemMetadata, Function,
-            EndOfInput
-        };
+            Comma,
```

Comment:
Team triage: The original version was grouped logically; can you insert some spaces between those sets to make it easier to follow?